base/src/main/java/proguard/optimize/inline/lambda_locator/LambdaLocator.java

```diff
@@ -0,0 +1,129 @@
+package proguard.optimize.inline.lambda_locator;
```

Comment:
java package convention is all lowercase, no underscores. In this case `lambdalocator`